src/components/component.ts

```diff
@@ -533,20 +515,18 @@ module Plottable {
     }
 
     /**
-     * Returns the foreground selection for the Component
-     * (A selection covering the front of the Component)
-     *
+     * Gets the Selection containing the <g> in front of the visual elements of the Component.
```

Comment:
Maybe change to
"Gets the foreground of the Component.
The foreground is a Selection containing the <g> in front of the visual elements of the Component."